src/pages/todos.tsx

```diff
@@ -0,0 +1,10 @@
+import { TodoItem } from "~/components/todo-item";
+import { TODOS } from "~/domains/todo/constants";
+
+export default function Home() {
+  const todos = TODOS;
+
+  const todoList = todos.map((todo) => <TodoItem key={todo.id} todo={todo} />);
+
+  return <ul>{todoList}</ul>;
```

Comment:
```suggestion
  return <ul className="flex flex-col gap-5">{todoList}</ul>;
```

Better to use this className for wrapper and you don't need `mb-5` for each `TodoItem` anymore.